infrastructure/main.tf

```diff
@@ -0,0 +1,27 @@
+resource "random_string" "password" {
+  length = 32
+  special = true
+}
```

Comment:
never used, can be removed

Reply:
Thought so but I was referring the draft store config so kept it.Will remove it